CMakeLists.txt

```diff
@@ -518,15 +518,6 @@ find_package(Boost 1.61 COMPONENTS ${BOOST_COMPONENTS} REQUIRED)
 include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 include_directories(SYSTEM ${PROJECT_BINARY_DIR}/include)
 
-if (WITH_SYSTEM_BOOST)
```

Comment:
@rjfd from a first glance, yeah, it is indeed a cleanup. but it was a side effect of turning libcommon into a share lib. i forgot exactly why.